applications/integrationtests/mechanics/StructuralGradientDamage.cpp

```diff
@@ -220,21 +220,21 @@ BOOST_AUTO_TEST_CASE(Integrand2D)
 
 
     using namespace NuTo::Visualize;
-    PostProcess visu("./GradientDamageOut2D");
+    PostProcess visu("./GradientDamageOutDecreasing2D");
     visu.DefineVisualizer("GDM", cells, VoronoiHandler(VoronoiGeometryTriangle(integration)));
     visu.Add("GDM", d);
     visu.Add("GDM", eeq);
     visu.Add("GDM", [&](const NuTo::CellIpData& data) { return gdm.mDamageLaw.Damage(gdm.Kappa(data)); }, "Damage");
 
     std::ofstream loadDisp(visu.ResultDirectory() + "/LD.dat");
 
-    auto doStep = [&](double t) { return problem.DoStep(t, "MumpsLU"); };
+    auto doStep = [&](double t) { return problem.DoStep(t, "EigenSparseLU"); };
     auto postProcess = [&](double t) {
         visu.Plot(t, true);
         problem.WriteTimeDofResidual(loadDisp, d, DofNumbering::Get(topNodes, ToComponentIndex(eDirection::Y)));
     };
 
     NuTo::AdaptiveSolve adaptive(doStep, postProcess);
     adaptive.dt = 0.01;
-    adaptive.Solve(adaptive.dt); // Only one step for this test. Increase to 1., if you want to see magic happen.
+    adaptive.Solve(1); // Only one step for this test. Increase to 1., if you want to see magic happen.
```

Comment:
Comment and code seem to disagree.

Reply:
Thanks! Just saw that travis debug took 10 minutes or so. This is the reason.